challenges/CC_29_SmartRockets/dna.js

```diff
@@ -4,31 +4,43 @@
 // Code for: https://youtu.be/bGz7mv2vD6g
 
 function DNA(genes) {
+  //recieves genes and create a dna object 
```

Comment:
Small thing but I prefer a comment style which starts with a capital letter and has a space after `//` like:

```javascript
// Receives genes and creates a dna object
```